package.json

```diff
@@ -19,6 +19,7 @@
     "classnames": "^2.3.2",
     "eslint": "8.33.0",
     "eslint-config-next": "13.1.6",
+    "jodit-react": "^1.3.10",
```

Comment:
Please run `npm install` once and commit the changes the happen to the `package-lock.json`. Currently they are out of sync in this branch.